2017/gonumtour.slide

```diff
@@ -0,0 +1,304 @@
+An Introduction to Gonum
+15 July 2017
+
+Brendan Tracey
+
+* Talk Outline
+- Overview of Gonum 
+- Gonum Philosophy
+- Brief examples of packages
+- Call for help
+
+* Overview of Gonum
+
+* What
+- Set of libraries for scientific and numeric programming (think: Numpy / Scipy)
+- Our packages (July 2017):
+
+.image gonumtour/packages.png 450 _
+
+* Where
+[[https://gonum.org][gonum.org]]
+- New [[https://gohugo.io/][Hugo]]-based site coming soon!
+
+	go get -u -t gonum.org/v1/gonum
+
+* Who
+To date: 35 Contributors
+
+Main contributors in New Mexico USA, Adelaide Australia, Karlsruhe Germany, Clermont-Ferrand France...
+
+That is, we have self-assembled. Come join us!
+
+- [[https://github.com/gonum][github.com/gonum]]
+- [[https://github.com/gonum/gonum/blob/master/CONTRIBUTING.md][Contributing guide]]
+- [[https://groups.google.com/forum/#!forum/gonum-dev][Mailing list]]
+
+* Goals of Gonum
+
+* Challenge: The Two Language Problem
+- Code that runs fast (C, C++, Fortran)
+- Code that is easy to write (Matlab, R, Python)
+- Original Matlab code snippet from a project
+
+	% Direct Method
```

Comment:
Deleting this line improves the layout, and it's already clear from the line above.